src/components/atoms/BoxSimple/BoxSimple.scss

```diff
@@ -19,6 +27,9 @@
     font-weight: 600;
     line-height: 33px;
     color: $gondola;
+    @media screen and (max-width: 768px) {
```

Comment:
What do you think, if we use a variable here?

For example, in a separate file, example breakpoints.scss

$breakpoint-tablet: 768px;

and after that

@import "breakpoints";
@media (min-width: $breakpoint-tablet) {
  
}

Reply:
I agree with using a variable for the breakpoints. I can make that change if everyone else agrees. 

I was second guessing how we should handle the media queries as well. I kept them inside the existing selectors because my thought was that it was easier to stay in the context of the selector I was working with.

Reply:
+1 to using variables for breakpoint values